testing/go/pgcatalog_test.go

```diff
@@ -172,12 +171,60 @@ func TestPgAttribute(t *testing.T) {
 					 INNER JOIN "pg_catalog"."pg_attribute" "att"
 					 ON "att"."attrelid" = "con"."confrelid" AND "att"."attnum" = "con"."child";`,
 					Expected: []sql.Row{
-						{"testing2_pktesting_fkey", "public", "testing2"},
+						// TODO: not clear test is defined correctly, returns no rows
```

Comment:
You can remove this test. Was just trying to get a simpler example for the query below but I think I removed some of the setup that gets this to return a row

Reply:
Thanks